pkg/chunk/series_store.go

```diff
@@ -58,17 +58,23 @@ type seriesStore struct {
 func newSeriesStore(cfg StoreConfig, schema Schema, storage StorageClient) (Store, error) {
 	fetcher, err := NewChunkFetcher(cfg.CacheConfig, storage)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "create chunk fetcher")
```

Comment:
I generally put error.Wrap at the very leaf of where the error is returns, in our code.  Adding on half way down the stack isn't that useful.

Reply:
Hmm, I disagree. Usually adding it here let's us track if it's from the chunk fetcher or from cache creation (below). Why do you think it isn't that useful?